src/variables/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$budha-gold: #cc9900;
+$white: #ffffff;
+$wide-medium-screen: 1414px;
+$medium-screen: 1278px;
+$small-screen: 900px;
+$ash: #c6c3b9;
+$hokey-pokey: #c9a131;
+$black: #000000;
+$l-screen: 1660px;
+$budha-border: rgba(204, 153, 0, 0.3);
```

Comment:
usually we use

$max-xs-screen: ...
$max-sm-screen: ...
$max-md-screen: ...

$min-sm-screen: $max-xs-screen + 1;
$min-md-screen: $max-sm-screen + 1;
$min-lg-screen: $max-md-screen + 1;

Reply:
changed